.github/workflows/release.yml

```diff
@@ -192,7 +192,7 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
       - uses: actions/download-artifact@v3
         with:
-          name: miranum-create-append-element-templates
+          name: miranum-create-append-c7-element-templates
```

Comment:
I would like to release the bpmn-js plugin I created on *npm*. Do I just need to update the pipeline for this, or are there additional steps I need to follow?

Reply:
Yes the pipeline looks good. Additionally, you have to make sure that your libs creates a package.json file that can be published
If you want to test the pipeline you can trigger the release pipeline manually on this branch to see if it works